CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Note: Breaking changes between versions are indicated by "💥".
 - [Improvement] Make all commands considerably faster
 - [Improvement] Make it easier to override Mongodb connection parameters
 - [Bugfix] Add support for .woff and .woff2 font files in themes (thanks @mrtndwrd!)
+- [Bugfix] Run `apt update` before `apt install` when installing deps in the openedx Dockerfile
 
```

Comment:
Sorry to nitpick; can you please move your line in the "Unreleased" section above?

Reply:
Done.